hummingbot/strategy_v2/executors/position_executor/position_executor.py

```diff
@@ -280,7 +280,7 @@ async def control_shutdown_process(self):
 
         :return: None
         """
-        if math.isclose(self.open_filled_amount, self.close_filled_amount) or (self.open_filled_amount > self.close_filled_amount):
+        if math.isclose(self.open_filled_amount, self.close_filled_amount) or (self.close_filled_amount > self.open_filled_amount):
```

Comment:
Hey @cardosofede, I found that this code might produce a bug where it can't close the position executor when the open_filled_amount is bigger than close_filled_amount in a significant amount that isclose return false, do you think (self.close_filled_amount > self.open_filled_amount) is good fix to make sure the position is closed properly in the case the fill amount is a bit bigger than expected?

Reply:
Here is example:

```
2024-06-13 19:56:14,498 - 21191 - hummingbot.strategy_v2.executors.position_executor.position_executor - INFO - Waiting for close order to be filled --> Filled amount: 11.31491505331653 | Open amount: 11.31378367494903
2024-06-13 19:56:14,499 - 21191 - hummingbot.strategy_v2.executors.position_executor.position_executor - INFO - Waiting for close order to be filled --> Filled amount: 11.31944328345293 | Open amount: 11.31831145230769
```

Reply:
@nikspz let's test that this don't impact the centralized exchanges

Reply:
@nikspz any updates here?

Reply:
based on discussion, the next steps are:
- increase the rel_tol of the math isclose
- remove flag to make the market order the exact amount
- based on the solution test the connector in xrpl and centralized exchanges @nikspz